packages/mds-jurisdiction-service/server/handlers/jurisdiction-schema-validators.ts

```diff
@@ -0,0 +1,41 @@
+/*
+    Copyright 2019-2020 City of Los Angeles.
+
+    Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+
+        http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+ */
+
+import {
+  uuidSchema,
+  stringSchema,
+  timestampSchema,
+  ValidateSchema,
+  ValidationError
+} from '@mds-core/mds-schema-validators'
+import { UUID, Timestamp } from '@mds-core/mds-types'
+import { JurisdictionDomainModel } from 'packages/mds-jurisdiction-service/@types'
+
+export const ValidateJurisdiction = (jurisdiction: JurisdictionDomainModel): JurisdictionDomainModel => {
+  try {
```

Comment:
This seems like a pretty verbose way to validate things. Out of curiosity, why do you prefer to have a separate schema for each property instead of writing one schema for the whole Jurisdiction type (a la say, 'eventSchema') and passing that in?

Reply:
I landed here because it resulted in slightly better error messaging, but I agree... using a jurisdictionSchema is less code.